packages/react-components/react-input/src/components/Input/useInputStyles.ts

```diff
@@ -17,96 +18,108 @@ const fieldHeights = {
+    borderBottomRightRadius: tokens.borderRadiusMedium,
 
-      // Flat 2px border:
-      // By default borderBottom will cause little "horns" on the ends. The clipPath trims them off.
-      // (This could be done without trimming using `background: linear-gradient(...)`, but using
-      // borderBottom makes it easier for people to override the color if needed.)
-      ...shorthands.borderBottom('2px', 'solid', tokens.colorCompoundBrandStroke),
-      clipPath: 'inset(calc(100% - 2px) 0 0 0)',
+    // Flat 2px border:
+    // By default borderBottom will cause little "horns" on the ends. The clipPath trims them off.
+    // (This could be done without trimming using `background: linear-gradient(...)`, but using
+    // borderBottom makes it easier for people to override the color if needed.)
+    borderBottom: `2px solid ${tokens.colorCompoundBrandStroke}`,
+    clipPath: 'inset(calc(100% - 2px) 0 0 0)',
 
-      // Animation for focus OUT
-      transform: 'scaleX(0)',
-      transitionProperty: 'transform',
-      transitionDuration: tokens.durationUltraFast,
-      transitionDelay: tokens.curveAccelerateMid,
+    // Animation for focus OUT
+    transform: 'scaleX(0)',
+    transitionProperty: 'transform',
+    transitionDuration: tokens.durationUltraFast,
+    transitionDelay: tokens.curveAccelerateMid,
 
-      '@media screen and (prefers-reduced-motion: reduce)': {
-        transitionDuration: '0.01ms',
-        transitionDelay: '0.01ms',
-      },
+    '@media screen and (prefers-reduced-motion: reduce)': {
+      transitionDuration: '0.01ms',
+      transitionDelay: '0.01ms',
     },
-    ':focus-within::after': {
-      // Animation for focus IN
-      transform: 'scaleX(1)',
-      transitionProperty: 'transform',
-      transitionDuration: tokens.durationNormal,
-      transitionDelay: tokens.curveDecelerateMid,
+  },
+  ':focus-within::after': {
+    // Animation for focus IN
+    transform: 'scaleX(1)',
+    transitionProperty: 'transform',
+    transitionDuration: tokens.durationNormal,
+    transitionDelay: tokens.curveDecelerateMid,
 
-      '@media screen and (prefers-reduced-motion: reduce)': {
-        transitionDuration: '0.01ms',
-        transitionDelay: '0.01ms',
-      },
-    },
-    ':focus-within:active::after': {
-      // This is if the user clicks the field again while it's already focused
-      borderBottomColor: tokens.colorCompoundBrandStrokePressed,
-    },
-    ':focus-within': {
-      outlineWidth: '2px',
-      outlineStyle: 'solid',
-      outlineColor: 'transparent',
+    '@media screen and (prefers-reduced-motion: reduce)': {
+      transitionDuration: '0.01ms',
+      transitionDelay: '0.01ms',
     },
   },
+  ':focus-within:active::after': {
+    // This is if the user clicks the field again while it's already focused
+    borderBottomColor: tokens.colorCompoundBrandStrokePressed,
+  },
+  ':focus-within': {
+    outlineWidth: '2px',
+    outlineStyle: 'solid',
+    outlineColor: 'transparent',
```

Comment:
_nit:_ You can reduce this to one line

```suggestion
    outline: '2px solid transparent',
```